src/server.ts

```diff
@@ -0,0 +1,11 @@
+import Express from 'express'
+import App from './app'
+import * as http from 'http'
+
+const PORT: string | number = process.env.PORT || 3131
```

Comment:
what if process.env.PORT is not there? it will return undefined i guess and not assignable to `string | number`, not sure, please confirm

Reply:
it will return the number in this case since will first evaluate the `env.PORT` if not found it will assign `3131` to the PORT constant.

Reply:
If process.env.PORT is not there, it will use 3131 (which's `number`)
`string | number` is used because typescript detects the env variables as strings.

Reply:
shit i missed that. I feel so stupid